### DISCUSSION TITLE
Partnership With Bitkeep wallet

### DISCUSSION THREAD

**Discussion by Soham66** [WalletConnect/walletconnect-monorepo#1145] [Developer Support]
Hello Wallet connect team , I am Soham - Business Developer of Bitkeep wallet

Bitkeep Wallet is the largest Multichain wallet In Asia and Top Growing in the World 

Bitkeep is verified on Telegram (130k+) And also verified on Twitter (210k+) 
Our Twitter :- https://twitter.com/BitKeepOS

I have a Valuable and Emerging Partnership proposal for your project 
Whom should I contact For partnership proposal ?

**Comment by hackyguru** [comment]
Hello Soham,
Thanks for your interest. You can mail the proposal to guru@walletconnect.com , I can make sure that the proposal reaches the right person.

**Reply by Soham66** [reply]
Hi guru , I think we already have Discussion on Discord and LinkedIn as well
